internal/logging/logging.go

```diff
@@ -239,11 +239,15 @@ func writeMessageDepth(depth int, level string, msg string, args ...interface{})
 	// We go over the args, and replace any function pointer with the signature
 	// func() interface{} with the return value of executing it now.
 	// This allows lazy evaluation of arguments which are return values
+	// Also, unpack error objects.
 	for i, arg := range args {
 		switch arg := arg.(type) {
 		case func() interface{}:
 			args[i] = arg
 		default:
+			if err, ok := arg.(error); ok {
+				args[i] = errs.JoinMessage(err)
+			}
```

Comment:
This switch is already a type-check, so I think it makes sense to move this to an `case` rather than use the default.

Reply:
You are correct. I was thinking if the error was something like a `locale.WrappedError`, the `arg.(type)` would not go into a `case error`. However, the debugger proved otherwise and I learned something new. Thanks for pointing this out!